src/main/scala/viper/gobra/backend/BackendVerifier.scala

```diff
@@ -49,7 +49,7 @@ object BackendVerifier {
 
     val verificationResults: Future[VerificationResult] =  {
       val verifier = config.backend.create(exePaths, config)
-      val reporter = BacktranslatingReporter(config.reporter, task.backtrack, config)
+      val reporter = BacktranslatingReporter(StreamingReporter(config.reporter), task.backtrack, config)
```

Comment:
@Dspil Would it be possible to set the `StreamingReporter` in the config instead of wrapping the reporter here?
What I image is that the reporter (or default reporter) for Gobra via CLI is the streaming reporter (maybe streaming reporter wrapping the current reporter). In this way, Gobra-IDE could use the existing reporter (i.e. a different one) without observing any changes